### PR TITLE
Add Make a HTTP Request Page

### PR DESCRIPTION
**Pull request by jennifertrin**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [X] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Make an HTTP Request
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:1]:
```suggestion
# Make an HTTP request
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Overview 
+
+Canister smart contracts can make a request to a URL using [HTTPS Outcalls](https://internetcomputer.org/https-outcalls). Canister smart contracts can make HTTPS outcalls to specified URLs to directly obtain off-chain data, or to interact with off-chain systems, such as Web 2.0 services or enterprise IT infrastructure.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:5]:
```suggestion
Canister smart contracts can make a request to a URL using [HTTPS outcalls](https://internetcomputer.org/https-outcalls). HTTPS outcalls can be used to directly obtain off-chain data or interact with off-chain systems, such as Web 2.0 services or enterprise IT infrastructure.
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+An HTTPS outcall mechanism for smart contracts enables a wide range of applications. For example, oracle services can directly integrate with ICP in a trustless manner, and dapps can provide user notifications and anything else that requires communication with servers on the Internet.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:7]:
```suggestion
HTTPS outcalls enable a wide range of applications. For example, oracle services can directly integrate with ICP in a trustless manner, and dapps can provide user notifications or anything else that requires communication with servers on the Internet.
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## How It Works
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:9]:
```suggestion
## How it works
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+For detailed information on how it works, please refer to the [HTTPS Outcalls - How It Works](./https-outcalls-how-it-works.md)
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:11]:
```suggestion
For detailed information on how it works, please refer to the [HTTPS outcalls - How it works](./https-outcalls-how-it-works.md).
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+## How to Make a Request
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:13]:
```suggestion
## How to make a request
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+For detailed information on how to utilize this feature, please refer to the [HTTPS Outcalls - How to Use](./https-outcalls-how-to-use.md) page.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-overview.md:15]:
```suggestion
For detailed information on how to utilize this feature, please refer to the [HTTPS outcalls - How to use](./https-outcalls-how-to-use.md) page.
```

File: sidebars.js [hunk 8]
```diff
@@ -368,6 +368,15 @@ const sidebars = {
                     "developer-docs/integrations/vetkeys/using-vetkeys",
                   ],
                 },
+                {
+                  type: "category",
+                  label: "HTTP Requests",
```
Comment (jessiemongeon1) [inline sidebars.js:373]:
```suggestion
                  label: "Making HTTP requests",
```

**Review by jessiemongeon1 — APPROVED** [review]

**Comment by jennifertrin** [comment]
Thank you for the review!

**Pull request merged**